skimage/feature/peak.py

```diff
@@ -11,8 +11,8 @@ def peak_local_max(image, min_distance=1, threshold_abs=None,
     Peaks are the local maxima in a region of `2 * min_distance + 1`
     (i.e. peaks are separated by at least `min_distance`).
 
-    If peaks are flat (i.e. multiple adjacent pixels have identical
-    intensities), the coordinates of all such pixels are returned.
+    If multiple peaks in the specified region have identical intensities,
+    the coordinates of all such pixels are returned.
```

Comment:
@jni not sure. See the link in my previous comment: which pixel has to be selected as the peak and why?
With the fresh view I can see now that the behaviour isn't really that bad, rather just ill-documented.